sdk/tables/azure-data-tables/azure/data/tables/aio/_base_client_async.py

```diff
@@ -19,26 +27,189 @@
     CustomHookPolicy,
     NetworkTraceLoggingPolicy,
 )
-from azure.core.pipeline.transport import (
-    AsyncHttpTransport,
-    HttpRequest,
-)
 
 from ._authentication_async import _configure_credential
+from .._common_conversion import _is_cosmos_endpoint
+from .._constants import DEFAULT_STORAGE_ENDPOINT_SUFFIX
 from .._generated.aio import AzureTable
-from .._base_client import AccountHostsMixin, get_api_version, extract_batch_part_metadata
+from .._base_client import extract_batch_part_metadata, parse_query, format_query_string, get_api_version
 from .._error import (
     RequestTooLargeError,
     TableTransactionError,
     _decode_error,
     _validate_tablename_error,
 )
-from .._policies import StorageHosts, StorageHeadersPolicy
+from .._models import LocationMode
+from .._policies import CosmosPatchTransformPolicy, StorageHeadersPolicy, StorageHosts
 from .._sdk_moniker import SDK_MONIKER
 from ._policies_async import AsyncTablesRetryPolicy
 
 
-class AsyncTablesBaseClient(AccountHostsMixin):
+class AsyncAccountHostsMixin(object):  # pylint: disable=too-many-instance-attributes
```

Comment:
```suggestion
class AsyncAccountHostsMixin:  # pylint: disable=too-many-instance-attributes
```

Reply:
If we are duplicating this class between sync and async - we actually no longer need it at all.
The entire `AccountHostsMixin` can be merged into `TablesBaseClient` and `AsyncTablesBaseClient`

Reply:
Merged AccountHostsMixin into TablesBaseClient and AsyncTablesBaseClient